.github/workflows/ci.yml

```diff
@@ -188,14 +188,14 @@ jobs:
         run: cmake --build ${{ matrix.build-dir }} --target check
 
       - name: Skeleton Validation
-        if: startsWith(runner.os, 'Windows') == false &&  matrix.skeleton == true
+        if: startsWith(runner.os, 'Windows') == false && matrix.skeleton == true
         working-directory: ${{ matrix.build-dir }}
         run: ./run_tests.sh --skeleton
 
       - name: Install
         run: |
-          cmake --build ${{ matrix.build-dir }} --target install
-          find install
+          cmake --build ${{ matrix.build-dir }} --config ${{ matrix.build-type }} --target install
+          echo -e "Intall tree:\n$(find install | sed 's/^/-- /')"
```

Comment:
This is part of the previous commit https://github.com/skvadrik/re2c/pull/319/commits/986af5bebb0552d5a39f825385eb73349ab977bf, right? Can you move it there?

Also, there is a small typo: `Intall` (missing "s").

Reply:
I'll squash commits at the end.  Thank you for the missing 's'!